UnityProject/Assets/Scripts/UI/Objects/GUI_PizzaBomb.cs

```diff
@@ -0,0 +1,130 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Data;
+using Items.Storage;
+using UnityEngine.UI;
+
+namespace UI.Objects
+{
+	public class GUI_PizzaBomb : NetTab
```

Comment:
it's no difference between C4 and pizza bomb?

Reply:
> it's no difference between C4 and pizza bomb?

It's a bit different because the C4/X4 is designed differently

Reply:
can you just, make it so that you get a pizza box and put a C4/x4 into it?
or is there extra functionality to pizza bombs I'm missing

Reply:
We talked about that already. I think that's bad for balance.